staking/role/commission.go

```diff
@@ -1,4 +1,4 @@
-package types
+package role
```

Comment:
what does role package mean? seems to me the files under "role" is not related to role (e.g. commission, decimal etc.)

Reply:
role means what part to play, that is, a validator is an entity, and nodes play the role of validator. Like I play the role of Banker, and part of my role is to have commissions